metadata-io/src/main/java/com/linkedin/metadata/entity/EntityService.java

```diff
@@ -586,20 +616,44 @@ public void ingestEntity(@Nonnull Entity entity, @Nonnull AuditStamp auditStamp,
             .collect(Collectors.toList())));
   }
 
+  /**
+  Returns true if entityType should have some aspect as per its definition
```

Comment:
This is still somewhat misleading in the fact that an entity may have an aspect like "Ownership" but it's not always _required_ to be provided in the set of aspects

Reply:
We could call this something like "isAspectProvided"

Reply:
will change to that